ArxContainer.h

```diff
@@ -704,10 +704,10 @@ struct deque : public RingBuffer<T, N> {
     using RingBuffer<T, N>::capacity;
     using RingBuffer<T, N>::pop;
     using RingBuffer<T, N>::push;
-    using RingBuffer<T, N>::emplace;
-    using RingBuffer<T, N>::assign;
-    using RingBuffer<T, N>::begin;
-    using RingBuffer<T, N>::end;
+    // NO! deque DOES have this! // using RingBuffer<T, N>::emplace;
+    // NO! deque DOES have this! // using RingBuffer<T, N>::assign;
+    // NO! deque DOES have this! // using RingBuffer<T, N>::begin;
+    // NO! deque DOES have this! // using RingBuffer<T, N>::end;
```

Comment:
You don't need to add these comments. Could you remove these lines?

Reply:
Those changes, exactly as coded, were clearly visible in the issue I raised, when you then asked me to create a PR. Did you not bother to look at the code at that time?
Then, after I create the PR (which incidentally allows edits by yourself), you ask me to change the code. Why didn't you ask me BEFORE I created the PR? At that time I would have done.
But now, as an incentive for you to consider reading things before asking others to do the work ... I will leave it to stand.
You are welcome to change the code as you wish.

Reply:
Haha, of course, I saw that, but I didn't think I would commit that comment as is. I'm sorry. This is not a job, but a volunteer activity of mine, so I hope you will tolerate it as a free activity between my regular work. I'll make my own PR then and merge it in.